configuration/exodus-storage.yaml

```diff
@@ -0,0 +1,91 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Configuration for exodus-cdn storage resources
+
+Parameters:
+  env:
+    Type: String
+    AllowedValues:
+      - dev
+      - stage
+      - prod
+    Default: dev
+    Description: The environment for which these resources are intended
+
+Resources:
+  Table:
+    Type: AWS::DynamoDB::Table
+    DeletionPolicy: Retain
+    Properties:
+      TableName: !Sub exodus-cdn-${env}
+      AttributeDefinitions:
+        -
+          AttributeName: web_uri
+          AttributeType: S
+        -
+          AttributeName: from_date
+          AttributeType: S
+      KeySchema:
+        -
+          AttributeName: web_uri
+          KeyType: HASH
+        -
+          AttributeName: from_date
+          KeyType: RANGE
+      BillingMode: PROVISIONED
+      ProvisionedThroughput:
+        ReadCapacityUnits: 10
```

Comment:
I reckon these may end up parametrized as well.  Dev environment should probably have far less capacity units than prod, for example, to avoid unnecessary costs.